server/lib/scene/scene.update.js

```diff
@@ -26,6 +34,21 @@ async function update(selector, scene) {
 
   await existingScene.update(scene);
 
+  await db.TagScene.destroy({
+    where: {
+      scene_id: existingScene.id,
+    },
+  });
+
+  if (scene.tags) {
+    await Promise.each(scene.tags, (tag) => {
```

Comment:
I think you could use bulkCreate for that

https://sequelize.org/docs/v6/core-concepts/model-querying-basics/